rope/contrib/autoimport/sqlite.py

```diff
@@ -529,6 +529,7 @@ def _get_packages_from_cache(self) -> List[str]:
             chain(*self._execute(models.Package.objects.select_star()).fetchall())
         )
         existing.append(self.project_package.name)
+        existing.append("site-packages")
```

Comment:
Happy to consolidate L531 and L532 into one
```suggestion
        existing += [self.project_package.name, "site-packages"]
```
but actually I like two `append`s better